src/py/flwr/server/superlink/state/state_test.py

```diff
@@ -409,6 +413,16 @@ def test_num_task_res(self) -> None:
         # Assert
         assert num == 2
 
+    def test_client_public_keys(self) -> None:
```

Comment:
In addition to this test, we need tests for all other new `State` methods.

Reply:
Added tests.